pydoro/pydoro_tui.py

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env python
 import threading
 
+import argparse
```

Comment:
isort this. ArgParse should be at the top of the file.

Reply:
Indeed. Seems like PyCharm did it for me 👍